typescript/sdk/src/token/EvmERC20WarpModule.hardhat-test.ts

```
@@ -972,14 +979,14 @@
         expect(check[0]).to.eql(allowedBridgeToAdd);
 
-        const allowance = await token.callStatic.allowance(
+        const allowance = await feeToken.callStatic.allowance(
           evmERC20WarpModule.serialize().deployedTokenRoute,
           allowedBridgeToAdd,
         );
         expect(allowance.toBigInt() === UINT_256_MAX).to.be.true;
       });
 
-      it(`should remove rebalancing bridges for tokens of type "${tokenType}"`, async () => {
+      it.only(`should remove rebalancing bridges for tokens of type "${tokenType}"`, async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)